cdap-docs/admin-manual/source/installation/installation.rst

```diff
@@ -618,13 +618,19 @@ you have a use case for it, please reach out to us at `cdap-user@googlegroups.co
 
        $ sudo yum install cdap cdap-gateway \
              cdap-hbase-compat-0.94 cdap-hbase-compat-0.96 cdap-hbase-compat-0.98 \
-             cdap-kafka cdap-master cdap-security cdap-web-app
+             cdap-kafka cdap-master cdap-security cdap-ui
 
    - Using APT (on one line)::
 
        $ sudo apt-get install cdap cdap-gateway \
              cdap-hbase-compat-0.94 cdap-hbase-compat-0.96 cdap-hbase-compat-0.98 \
-             cdap-kafka cdap-master cdap-security cdap-web-app
+             cdap-kafka cdap-master cdap-security cdap-ui
+
+#  Remove link to older cdap-web-app service
+       $ rm -rf /etc/init.d/cdap-web-app
+
+   We have deprecated the cdap-web-app package. So the links to older cdap-web-app service
+   should be removed
```

Comment:
I had the following:

Note that the ``cdap-ui`` package is new in CDAP 3.0.0, and replaces the ``cdap-web-app`` package.
On some Ubuntu systems, the upgrade may leave the ``cdap-web-app`` package in an ``ic`` state, meaning only its
configuration files are installed.  It is safe to completely remove this package, using ``apt-get purge cdap-web-app``